nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/common/BooleanLatch.java

```diff
@@ -1,66 +0,0 @@
-/*
```

Comment:
This class was completely unused - I assume the version of this class in another module is the copy-pasted version of this.